README Rework (#173)
LucidVR/opengloves-driver: README.md

# OpenGloves Driver [![Steam Badge]][Steam] [![Discord Badge]][Discord]

> A general purpose **SteamVR** driver for **DIY** VR <br>
> Glove hardware providing an **[Open Interface]**.

Reviewer: ```suggestion
A general purpose **SteamVR** driver for **DIY** VR <br>
(Glove) hardware, which provides an **[Open Interface]** for anyone wanting to connect their own hardware to SteamVR,
```

---

**⸢ [Installation] ⸥ ⸢ [Limitations] ⸥ ⸢ [Compatibility] ⸥**

---

## Features

- **Haptic Force Feedback**

- **Full Finger Tracking** <br>
    *‹ Beta › Splay Support* <br>
    *‹ Beta › Individual Joint Support*

- **Tracker / Controller Positioning** <br>
    *Automatic relative calibration* <br>
    *Controller ⟷ Glove*

- **Button / Joystick Inputs** <br>
    *➜ Trigger | A | B* <br>

>     *➜ Joystick | X | Y*

Reviewer: joysticks also have a click
```suggestion
    *➜ Joystick | X | Y | Click*
```

- **Multiple Communication Protocols** <br>
    *- Bluetooth Serial* <br>
    *- Named Pipes* <br>
    *- Serial USB*

<br>

### Steam UI

> The **[Steam]** version of **OpenGloves** also provides you a **[UI]**.

Reviewer: Perhaps?
```suggestion
When **OpenGloves** is downloaded on **[Steam]**, it comes with a **[UI]** for configuring driver related features.
```
